[2024-09-21T21:02:09.620+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.spark_raw_ingestion_previsao manual__2024-09-21T21:02:08.524614+00:00 [queued]>
[2024-09-21T21:02:09.629+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.spark_raw_ingestion_previsao manual__2024-09-21T21:02:08.524614+00:00 [queued]>
[2024-09-21T21:02:09.629+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 0
[2024-09-21T21:02:09.643+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): spark_raw_ingestion_previsao> on 2024-09-21 21:02:08.524614+00:00
[2024-09-21T21:02:09.647+0000] {standard_task_runner.py:57} INFO - Started process 583 to run task
[2024-09-21T21:02:09.650+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'spark_raw_ingestion_previsao', 'manual__2024-09-21T21:02:08.524614+00:00', '--job-id', '358', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmp3tejyzyl']
[2024-09-21T21:02:09.651+0000] {standard_task_runner.py:85} INFO - Job 358: Subtask spark_raw_ingestion_previsao
[2024-09-21T21:02:09.773+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.spark_raw_ingestion_previsao manual__2024-09-21T21:02:08.524614+00:00 [running]> on host airflow-worker
[2024-09-21T21:02:09.973+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='spark_raw_ingestion_previsao' AIRFLOW_CTX_EXECUTION_DATE='2024-09-21T21:02:08.524614+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-21T21:02:08.524614+00:00'
[2024-09-21T21:02:09.974+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-21T21:02:09.974+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-21T21:02:10.005+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-21T21:02:10.006+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-21T21:02:10.014+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-21T21:02:10.105+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-21T21:02:10.105+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g --conf "spark.pyspark.python=/env/venv/bin/python" /opt/shared/scripts/raw_ingestion.py 2024-09-21 previsao
[2024-09-21T21:02:12.548+0000] {ssh.py:524} INFO - Traceback (most recent call last):
[2024-09-21T21:02:12.548+0000] {ssh.py:524} INFO -   File "/opt/shared/scripts/raw_ingestion.py", line 1, in <module>
[2024-09-21T21:02:12.548+0000] {ssh.py:524} INFO -     import requests, json, time, sys
[2024-09-21T21:02:12.549+0000] {ssh.py:524} INFO - ModuleNotFoundError: No module named 'requests'
[2024-09-21T21:02:12.556+0000] {ssh.py:524} INFO - 24/09/21 21:02:12 INFO ShutdownHookManager: Shutdown hook called
[2024-09-21T21:02:12.562+0000] {ssh.py:524} INFO - 24/09/21 21:02:12 INFO ShutdownHookManager: Deleting directory /tmp/spark-1d7bc533-a893-4a53-9fb4-6b16c54c61c9
[2024-09-21T21:02:12.674+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 166, in run_ssh_client_command
    self.raise_for_status(exit_status, agg_stderr, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 160, in raise_for_status
    raise AirflowException(f"SSH operator error: exit status = {exit_status}")
airflow.exceptions.AirflowException: SSH operator error: exit status = 1
[2024-09-21T21:02:12.678+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion, task_id=spark_raw_ingestion_previsao, execution_date=20240921T210208, start_date=20240921T210209, end_date=20240921T210212
[2024-09-21T21:02:12.707+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 358 for task spark_raw_ingestion_previsao (SSH operator error: exit status = 1; 583)
[2024-09-21T21:02:12.754+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-21T21:02:12.796+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
